.env.example

```diff
@@ -6,5 +6,5 @@
 
 CHROMATIC_PROJECT_TOKEN=<place app_code here>
 
-# CHROMATIC_INDEX_URL=https://www.chromaticqa.com
-# CHROMATIC_TUNNEL_URL=http://tunnel.chromaticqa.com
+# CHROMATIC_INDEX_URL=https://www.chromatic.com
+# CHROMATIC_TUNNEL_URL=http://tunnel.chromatic.com
```

Comment:
The tunnel still uses `tunnel.chromaticqa.com` as `tunnel.chromatic.com` doesn't actually exist.